seahorn/CMakeLists.txt

```diff
@@ -251,9 +251,12 @@ macro(sea_add_klee_bc name)
   set(TARGET_KLEE "${name}.klee")
   add_executable(${TARGET_KLEE} ${ARG_UNPARSED_ARGUMENTS})
   target_compile_definitions(${TARGET_KLEE} PUBLIC 
-    MAX_INITIAL_SIZE=${MAX_INITIAL_SIZE}
+    MAX_BUFFER_SIZE=${MAX_BUFFER_SIZE}
```

Comment:
none of these size definitions are needed. They make it very hard to override the sizes later on.
Instead, code should include `config.h` and then inherit the definitions from the original target.